File: app/src/main/java/com/chooloo/www/callmanager/ui/recents/RecentsMvpView.java
package com.chooloo.www.callmanager.ui.recents;

import androidx.annotation.Nullable;

import com.chooloo.www.callmanager.entity.RecentCall;
import com.chooloo.www.callmanager.ui.cursor.CursorMvpView;

public interface RecentsMvpView extends CursorMvpView {
    void openRecent(RecentCall recentCall);

    int getItemCount();

    void load(@Nullable String phoneNumber, @Nullable String contactName);
}

File: app/src/main/java/com/chooloo/www/callmanager/ui/recents/RecentsMvpView.kt
package com.chooloo.www.callmanager.ui.recents

import com.chooloo.www.callmanager.entity.Recent
import com.chooloo.www.callmanager.ui.cursor.CursorMvpView

interface RecentsMvpView : CursorMvpView {
    override val itemCount: Int
    fun openRecent(recent: Recent)
}